tasks/stages/update-demo

```diff
@@ -1,5 +1,8 @@
 #! /bin/bash
 
+set -euo pipefail
```

Comment:
Is it necessary? https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#exit-codes-and-error-action-preference

Reply:
It is, as I can run the same script locally. In that case it should exhibit the same behavior, too.